README.md

```diff
@@ -72,6 +72,23 @@ You can access the documentation online, which is hosted on [Read the Docs](http
 * [FastRTPSGen manual](http://eprosima-fast-rtps.readthedocs.io/en/latest/geninfo.html)
 * [Release notes](http://eprosima-fast-rtps.readthedocs.io/en/latest/notes.html)
 
+## Quick Demo
+
+For those who want to try a quick demonstration of Fast-RTPS libraries, here is a way to launch an example application.
+
+For that, it is necessary to download **docker** application. In Ubuntu, open a terminal and type the following command
+
+	$ sudo apt-get install docker.io
+
+Then, you have to download the docker image file from https://eprosima.com/index.php/downloads-all
+
+Load the image and run it
+
+	$ docker load -i ubuntu-fast-rtps.tar
+	$ docker run -it ubuntu-fast-rtps
+
+You can run as many images as you want and check the communication between them
+
```

Comment:
Please copy this section from https://github.com/eProsima/Fast-RTPS/pull/584/files and make sure it is exactly the same.